config/environments/production.rb

```diff
@@ -88,10 +73,31 @@
   if ENV['RAILS_LOG_TO_STDOUT'].present?
     logger           = ActiveSupport::Logger.new(STDOUT)
     logger.formatter = config.log_formatter
-    config.log_tags  = %i[subdomain uuid]
+    config.log_tags  = %i[subdomain uuid] # commented out with Rails 6.1 upgrade
```

Comment:
This doesn't look commented out.

Reply:
good eyes - removing comment.  The line was removed in Rails 6.1 upgrade, but I suspect it is a line we added deliberately so I'm keeping it.